unattended_scripts/wazuh_install.sh

```diff
@@ -211,44 +272,66 @@ main() {
     importFunction "wazuh-cert-tool.sh"
 
     checkArch
+
+    
     
     if [ -n "${certificates}" ] || [ -n "${AIO}" ]; then
         createCertificates
     fi
 
+    if [ -n "${elasticsearch}" ] && [ -n "${kibana}" ]; then
+        mult_install=1
+    fi
+    if [ -n "${elasticsearch}" ] && [ -n "${wazuh}" ]; then
+        mult_install=1
+    fi
+    if [ -n "${kibana}" ] && [ -n "${wazuh}" ]; then
+        mult_install=1
+    fi
+
     if [ -n "${elasticsearch}" ]; then
 
         importFunction "elasticsearch.sh"
 
+
+        progressbar_total=8
+        progressbar_status=0
         if [ -n "${ignore}" ]; then
             logger -w "Health-check ignored."
+            ((progressbar_status++))
         else
             healthCheck elasticsearch
         fi
         checkSystem
         installPrerequisites
         addWazuhrepo
         checkNodes
-        installElasticsearch 
+        installElasticsearch
         configureElasticsearch
         restoreWazuhrepo
+        logger "Elasticsearch installed correctly"
+
     fi
 
     if [ -n "${kibana}" ]; then
 
         importFunction "kibana.sh"
 
+        progressbar_total=8
```

Comment:
Same

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/1080/commits/b6529a2b3b0eeb287bd573b77284f34b2fef99a6